app/controllers/services/govbox_controller.rb

```diff
@@ -84,4 +85,8 @@ def ensure_live
       render template: 'errors/not_found', status: 404, layout: 'application'
     end
   end
+
+  def convert_mode
+    (params[:mode] == 'on') ? params[:mode] = 'api_mode' : params[:mode] = nil
```

Comment:
toto je velmi divna konstrukcia. Ternarny operator sa nepouziva na priradenie takto. Ked uz tak 
`params[:mode] = params[:mode] == 'on' ? 'api_mode' : nil`

zaroven by som toto uplne vyrazil a nastavil proste dobru value na tom checkboxe. after_action nedava zmysel, ma to byt asi before_action a zaroven kedze sa to pouziva len an jednom mieste to nema cenu vytahovat do hooku.

Reply:
neviem ako nastavit `value="api_mode"` s rails
`def check_box_tag(name, value = "1", checked = false, options = {})` s tym ze chcem `value` posielat do `params[:mode]`.

kedze volam ako `check_box_tag(:api_interest, params[:mode], class: 'mr-2')` tak neviem nastavit `value`

Reply:
A `check_box_tag(:mode, 'api'...)` nejde preco?

Reply:
Uz ide